src/control/cmd/daos_agent/main.go

```diff
@@ -281,6 +281,10 @@ func main() {
 	)
 
 	if err := parseOpts(os.Args[1:], &opts, ctlInvoker, log); err != nil {
+		if fe, ok := errors.Cause(err).(*flags.Error); ok && fe.Type == flags.ErrHelp {
```

Comment:
Maybe a candidate for a helper function in common? Or put some logic shared by our executables in the cmd package? I've run into a couple cases where there wasn't a great way to share the logic amongst binaries.

Reply:
There is a helper called `flags.WroteHelp()` in the master branch of the go-flags package that basically does this, but it isn't available in any released versions yet. I figured rather than adding a new helper that should go away eventually I'd just make it explicit and we can replace with the helper when go-flags is updated.